custom/ilsgateway/templates/ilsgateway/slab/edit_location.html

```diff
@@ -3,19 +3,20 @@
 {% load hq_shared_tags %}
 {% load crispy_forms_tags %}
 
-{% block stylesheets %} {{ block.super }}
-    <link rel="stylesheet" href="{% static 'hqwebapp/js/lib/jquery-ui/multiselect/ui.multiselect.css' %}" />
-{% endblock %}
-
 {% block js %}{{ block.super }}
     <script src="{% static 'style/js/bootstrap2/ui-element.js' %}"></script>
-    <script src="{% static 'hqwebapp/js/lib/jquery-ui/multiselect/ui.multiselect.js' %}"></script>
 {% endblock %}
 
 {% block js-inline %}{{ block.super }}
     <script type="text/javascript">
         $(function () {
-            $("#id_selected_ids").width(800).height(400).multiselect();
+            var multiselect_utils = hqImport('style/js/components/multiselect_utils');
+            multiselect_utils.createFullMultiselectWidget(
+                'id_selected_ids',
+                django.gettext("Available Locations"),
```

Comment:
@czue @kkrampa does this text make sense in the context of this widget? I'm not familiar with this UI to say 100% that this is a good word choice.